### ISSUE TITLE
[Drill]: Screen reader is staying silent, when 'Next' and 'Previous' buttons are activated.

### ISSUE THREAD

**Issue by msftedad** [grafana/grafana#86834]
### What happened?

Screen reader is not announcing the updated time zone when 'Next' and 'Previous' buttons are activated. 

Screen reader stays silent. 

### What did you expect to happen?

Screen reader should announce the updated time zone when 'Next' and 'Previous' buttons are activated. 

### Did this work before?

No

### How do we reproduce it?

1.Open the URL in edge browser [Grafana Play Home - Dashboards - Grafana](https://play.grafana.org/d/000000012/grafana-play-home?orgId=1). 

2.‘Grafana Play Home’ page will open. 

3.Navigate to ‘Time range’ button & expand it. 

4.Navigate to ‘From’ & ‘To’ fields, select respective dates from date picker dialog. 

5.Now, again navigate to ‘Forward’ & ‘backward’ buttons present for the time zone & activate them. 

6.Observe the screen reader narration. 

### Environment (with versions)?

Grafana: v11.1.0-69051 (9d44c8e8cf)
OS: Version 24H2 (OS Build 26100.1) 
Browser: Version 125.0.2506.0 (Official build) dev (64-bit) 

### Grafana platform?

None
